src/index.js

```diff
@@ -1,118 +1,40 @@
-/* eslint-disable max-classes-per-file */
-/* eslint-disable no-unused-vars */
-/* eslint-disable camelcase */
-
-import _ from 'lodash';
 import './style.css';
-import Icons from '../assets/more.png';
-import Icons2 from '../assets/refresh.png';
-
-class Actions {
-  constructor(des, com, nums) {
-    this.description = des;
-    this.completed = com;
-    this.index = nums;
-  }
-}
-
-class List {
-  constructor() {
-    this.Lists = [];
-  }
-
-  addToEnd(Des, Com, Nums) {
-    const NewAction = new Actions(Des, Com, Nums);
-    this.Lists.push(NewAction);
-  }
-
-  addToFront(Des, Com, Nums) {
-    const NewAction = new Actions(Des, Com, Nums);
-    this.Lists.unshift(NewAction);
-  }
-
-  removeFromFront() {
-    this.Lists.unshift();
-  }
-
-  removeFromBack() {
-    this.Lists.pop();
-  }
-
-  removeAt(index) {
-    this.Lists.splice(index, 1);
-  }
-
-  printing() {
-    const ans = [];
-    for (let x = 0; x < this.Lists.length; x += 1) {
-      ans.push(this.Lists[x]);
-    }
-    return ans;
-  }
+import { saveLocalStorage, updateCompleted } from './active';
+
+let items = [
+  {
+    description: 'wash the dishes',
+    completed: true,
+    index: 0,
```

Comment:
[OPTIONAL]

- I would recommend using a default value of completed is `false`. As this is a small change, and other than this everything looks fine. I will approve the PR.